src/recovery/mod.rs

```diff
@@ -124,6 +128,13 @@ pub struct Recovery {
 
     // HyStart++.
     hystart: hystart::Hystart,
+
+    // Pacing
+    pacing_enabled: bool,
+
+    pacing_rate: usize,
```

Comment:
Why `usize`? It seems like `u64` would be more appropriate.

Reply:
will take care

Reply:
what's the unit of pacing_rate?

Reply:
bytes/sec